### PR TITLE
cleanup(angular): replace usage of wrapAngularDevkitSchematics for application

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

We currently rely on running the Angular Application Generator from @schematics/angular first before we modify the output to suit Nx.

## Expected Behavior

We should generate what we need ourselves

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/generators/application/files/ng-module/src/app/app.component.spec.ts__tpl__ [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    expect(app.title).toEqual('<%= appName %>');
+  });
+  <% if(!minimal) { %>
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    fixture.detectChanges();
+    const compiled = fixture.nativeElement as HTMLElement;
+    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
+  });
+  <% } %>
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/ng-module/src/app/app.component.spec.ts__tpl__:32]: Let's remove the useless `should create the app` test case and rearrange the rest based on proposed changes to the component generation:
```suggestion
  it('should render title', () => {
    const fixture = TestBed.createComponent(AppComponent);
    fixture.detectChanges();
    const compiled = fixture.nativeElement as HTMLElement;
    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
  });<% if(!minimal) { %>

  it(`should have as title '<%= appName %>'`, () => {
    const fixture = TestBed.createComponent(AppComponent);
    const app = fixture.componentInstance;
    expect(app.title).toEqual('<%= appName %>');
  });<% } %>
```

File: packages/angular/src/generators/application/files/ng-module/src/app/app.component.ts__tpl__ [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  selector: '<%= rootSelector %>',<% if(!inlineTemplate) { %>
+  templateUrl: './app.component.html',<% } else { %>
+  template: `<% if(!minimal) { %><<%= nxWelcomeSelector %>></<%= nxWelcomeSelector %>><% } %> <% if(routing) { %><router-outlet></router-outlet><% } %>`,<% } %><% if(!inlineStyle) { %>
+  styleUrls: ['./app.component.<%= style %>'],<% } else { %>
+  styles: [''],<% } %><% if(viewEncapsulation) { %>
+  encapsulation: ViewEncapsulation.<%= viewEncapsulation %>,<% } %>
+})
+export class AppComponent {
+  title = '<%= appName %>';
+}
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/ng-module/src/app/app.component.ts__tpl__:13]: Let's remove the `title` variable for `minimal`:
```suggestion
export class AppComponent {<% if(!minimal) { %>
  title = '<%= appName %>';
<% } %>}
```

File: packages/angular/src/generators/application/files/standalone-components/src/app/app.component.spec.ts__tpl__ [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    expect(app.title).toEqual('<%= appName %>');
+  });
+  <% if(!minimal) { %>
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    fixture.detectChanges();
+    const compiled = fixture.nativeElement as HTMLElement;
+    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
+  });
+  <% } %>
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/app.component.spec.ts__tpl__:31]:
```suggestion
  it('should render title', () => {
    const fixture = TestBed.createComponent(AppComponent);
    fixture.detectChanges();
    const compiled = fixture.nativeElement as HTMLElement;
    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
  });<% if(!minimal) { %>
  it(`should have as title '<%= appName %>'`, () => {
    const fixture = TestBed.createComponent(AppComponent);
    const app = fixture.componentInstance;
    expect(app.title).toEqual('<%= appName %>');
  });<% } %>
```

File: packages/angular/src/generators/application/files/standalone-components/src/app/app.component.ts__tpl__ [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  selector: '<%= rootSelector %>',<% if(!inlineTemplate) { %>
+  templateUrl: './app.component.html',<% } else { %>
+  template: `<% if(!minimal) { %><<%= nxWelcomeSelector %>></<%= nxWelcomeSelector %>><% } %> <% if(routing) { %><router-outlet></router-outlet><% } %>`,<% } %><% if(!inlineStyle) { %>
+  styleUrls: ['./app.component.<%= style %>'],<% } else { %>
+  styles: [''],<% } %><% if(viewEncapsulation) { %>
+  encapsulation: ViewEncapsulation.<%= viewEncapsulation %>,<% } %>
+})
+export class AppComponent {
+  title = '<%= appName %>';
+}
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/app.component.ts__tpl__:17]:
```suggestion
export class AppComponent {<% if(!minimal) { %>
  title = '<%= appName %>';
<% } %>}
```

File: packages/angular/src/generators/application/files/ng-module/src/app/nx-welcome.component.ts__tpl__ [hunk 5]
```diff
@@ -281,10 +284,10 @@ export const nrwlHomeTemplate = {
       .button-pill:hover {
         color: rgba(255, 255, 255, 1) !important;
       }
-      .nx-console:hover {
+      #nx-console:hover {
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/ng-module/src/app/nx-welcome.component.ts__tpl__:287]: Let's avoid changes to the existing template, aside from formatting.

File: packages/angular/src/generators/application/files/standalone-components/src/app/nx-welcome.component.ts__tpl__
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/nx-welcome.component.ts__tpl__:1]: Check this as well.

File: packages/angular/src/generators/application/lib/create-project.ts [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+            namedChunks: true,
+          },
+        },
+        defaultConfiguration: 'production',
+      },
+      serve: {
+        executor: '@angular-devkit/build-angular:dev-server',
+        options: {
+          port: options.port ?? 4200,
+        },
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/lib/create-project.ts:63]:
```suggestion
        options: options.port
          ? {
            port: options.port,
          }
          : undefined,
```

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: packages/angular/src/generators/application/files/base/tsconfig.app.json__tpl__ [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "<%= rootOffset %>dist/out-tsc",
+    "types": []
+  },
+  "files": ["src/main.ts"],
+  "include": ["src/**/*.d.ts"],
+  "exclude": ["jest.config.ts", "src/**/*.test.ts", "src/**/*.spec.ts"]
```
[the remark this answers is not in the document]
Comment (Coly010) [reply packages/angular/src/generators/application/files/base/tsconfig.app.json__tpl__:9]: This is existing behaviour so we can address this in a future PR

File: packages/angular/src/generators/application/files/ng-module/src/app/app.component.spec.ts__tpl__ [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    }).compileComponents();
+  });
+
+  it('should render title', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    fixture.detectChanges();
+    const compiled = fixture.nativeElement as HTMLElement;
+    expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
+  });<% if(!minimal) { %>
+  it(`should have as title '<%= appName %>'`, () => {
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/ng-module/src/app/app.component.spec.ts__tpl__:20]:
```suggestion
  });<% if(!minimal) { %>

  it(`should have as title '<%= appName %>'`, () => {
```

File: packages/angular/src/generators/application/files/ng-module/src/main.ts__tpl__ [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
```
[the remark this answers is not in the document]
Comment (leosvelperez) [reply packages/angular/src/generators/application/files/ng-module/src/main.ts__tpl__:3]: This change is still missing

File: packages/angular/src/generators/application/files/standalone-components/src/app/app.component.spec.ts__tpl__ [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+    }).compileComponents();
+  });
+
+  it('should render title', () => {
+      const fixture = TestBed.createComponent(AppComponent);
+      fixture.detectChanges();
+      const compiled = fixture.nativeElement as HTMLElement;
+      expect(compiled.querySelector('h1')?.textContent).toContain('Welcome <%= appName %>');
+  });<% if(!minimal) { %>
+  it(`should have as title '<%= appName %>'`, () => {
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/app.component.spec.ts__tpl__:19]:
```suggestion
  });<% if(!minimal) { %>

  it(`should have as title '<%= appName %>'`, () => {
```

Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/app.component.spec.ts__tpl__:19]:
```suggestion
  });<% if(!minimal) { %>

  it(`should have as title '<%= appName %>'`, () => {
```

File: packages/angular/src/generators/application/files/standalone-components/src/app/nx-welcome.component.ts__tpl__ [hunk 11]
```diff
@@ -0,0 +1,810 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+/* eslint-disable */
```
Comment (leosvelperez) [inline packages/angular/src/generators/application/files/standalone-components/src/app/nx-welcome.component.ts__tpl__:4]: Remove

**Review by leosvelperez — CHANGES_REQUESTED** [review]
Some few minor things left

**Review by leosvelperez — APPROVED** [review]

**Pull request merged**